content/en/monitors/guide/how-to-set-up-rbac-for-monitors.md

```diff
@@ -121,21 +121,15 @@ For more information, see [Roles][5] and [Monitors API Reference][6] .
 {{% /tab %}}
 {{< /tabs >}}
 
-## Migrate monitors from locked to restricted roles
+## Restricted roles
 
-Before Datadog released the feature allowing restriction of monitor editing to specific roles, monitors could be locked. Only the creator and users with the [Datadog Admin Role][2] can edit a locked monitor. 
-
-{{< img src="/monitors/guide/monitor_rbac_locked.jpg" alt="RBAC Locked Monitor" style="width:70%;">}}
-
-Locked monitors are deprecated and are no longer supported. Instead, use the role restriction option, which gives you more flexibility to define which users are allowed to edit monitors.
-
-The sections below describe how to migrate from the locked mechanism to restricted roles, depending on the way you manage your monitors.
+Datadog allows restriction of monitor editing to specific roles through the role restriction option. This gives you the flexibility to define which users are allowed to edit monitors.
 
 ### API
 
-The `locked` parameter corresponding to the above mentioned locking mechanism is no longer supported. This means you must update the definition of your monitors managed through API or Terraform to stop using `locked` and start using `restricted_roles` (parameter attached with the new role restriction option).
+You can update the definition of your monitors managed through API or Terraform using `restricted_roles` (parameter attached with the new role restriction option).  You can also use the [Restriction Policies][5] endpoint to define the access control rules for a monitor, mapping a set of relations (such as editor and viewer) to a set of allowed principals (such as roles, teams, or users). The restriction policy determines who is authorized to perform what actions on the monitor.
```

Comment:
I'm not sure what the first sentence and its parenthetical are describing. Can you clarify?

Reply:
@neko-dd Removed the parenthetical. The `restricted_roles` parameter is for the [API](https://docs.datadoghq.com/api/latest/monitors/). The parenthetical should be removed since this option is the default, not a new feature anymore. Thanks!

```suggestion
You can update the definition of your monitors managed through API or Terraform with the `restricted_roles` parameter.  You can also use the [Restriction Policies][5] endpoint to define the access control rules for a monitor, mapping a set of relations (such as editor and viewer) to a set of allowed principals (such as roles, teams, or users). The restriction policy determines who is authorized to perform what actions on the monitor.
```